src/components/loginForm/LoginForm.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import "./LoginForm.css"; // Import your CSS file
+function LoginForm() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const handleSubmit = async (event) => {
+    event.preventDefault(); // Prevent default form submission
+    try {
+      const response = await fetch("/api/login", {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify({ email, password }),
+      });
+      if (response.status === 200) {
+        console.log("Login successful!");
+      } else {
+        console.error("Login failed");
+      }
+    } catch (error) {
+      console.error("Error during login:", error);
+    }
+  };
+  return (
+    <div className="login-container">
+      {/*<img src="logo.png" alt="ShareSource Logo" className="logo" />  Replace with actual logo */}
```

Comment:
cleanup commented code from all the files in your PR

Reply:
done